templates/endpoints_v2/README.ipynb

```diff
@@ -220,13 +265,18 @@
     "- Production-grade autoscaling support, including scale-to-zero.\n",
     "- Native multi-GPU & multi-node model deployments.\n",
     "\n",
-    "Look at the following guides for more advanced use-cases:\n",
-    "* [Deploy models for embedding generation](examples/embedding/EmbeddingModels.ipynb)\n",
-    "* [Learn how to bring your own models](examples/CustomModels.ipynb)\n",
-    "* [Deploy multiple LoRA fine-tuned models](examples/lora/DeployLora.ipynb)\n",
-    "* [Deploy Function calling models](examples/function_calling/DeployFunctionCalling.ipynb)\n",
-    "* [Learn how to leverage different configurations that can optimize the latency and throughput of your models](examples/OptimizeModels.ipynb)\n",
-    "* [Learn how to fully configure your deployment including auto-scaling, optimization parameters and tensor-parallelism](examples/AdvancedModelConfigs.ipynb)\n",
+    "## Cookbooks\n",
+    "\n",
+    "After you are done with the above, you can find recipies that extend the functionality of this template under the cookbooks folder:\n",
+    "\n",
+    "* [Deploy models for embedding generation](cookbooks/embedding/EmbeddingModels.ipynb)\n",
+    "* [Deploy multiple LoRA fine-tuned models](cookbooks/lora/DeployLora.ipynb)\n",
+    "* [Deploy Function calling models](cookbooks/function_calling/DeployFunctionCalling.ipynb)\n",
+    "* [Deploy Vision Language Model](cookbooks/vision_language_model/README.ipynb): Build a Gradio appliation on top of LLaVA-NeXT Mistral 7B.\n",
+    "* [Learn how to bring your own models](cookbooks/CustomModels.ipynb)\n",
+    "* [Learn how to leverage different configurations that can optimize the latency and throughput of your models](cookbooks/OptimizeModels.ipynb)\n",
+    "* [Learn how to fully configure your deployment including auto-scaling, optimization parameters and tensor-parallelism](cookbooks/AdvancedModelConfigs.ipynb)\n",
+    "\n",
```

Comment:
For now let's remove the Examples section and add it later, once we have content. I want us to clean up any existing remainder of Anyscale Endpoints first.

Reply:
Removed Application Examples.